draft-ietf-oauth-cross-device-security.md

```diff
@@ -282,9 +285,9 @@ Attackers exploit this absence of an authenticated channel between the two devic
            |              |     QR Code/User Code |               |
            |              |     Context           |               |
            +--------------+                       |               |
-                  | (E) User is tricked and       |               |
-                  |     Scan QR code or           |               |
-                  |     enter User Code           |               |
+                  | (E) User is convinced by the  |               |
+                  |     attacker and scan QR code |               |
+                  |     or enter User Code        |               |
```

Comment:
Not sure if this should be 'scans' and 'enters', but then we need to change "Starts" and "Copies" above as well...

Reply:
Made the proposed changes